api4/saml.go

```diff
@@ -247,3 +250,28 @@ func getSamlMetadataFromIdp(c *Context, w http.ResponseWriter, r *http.Request)
 
 	w.Write([]byte(metadata.ToJson()))
 }
+
+func resetAuthDataToEmail(c *Context, w http.ResponseWriter, r *http.Request) {
+	if !c.App.SessionHasPermissionTo(*c.App.Session(), model.PERMISSION_MANAGE_SYSTEM) {
+		c.SetPermissionError(model.PERMISSION_MANAGE_SYSTEM)
+		return
+	}
```

Comment:
Do we need a license check here?

Reply:
None of the other SAML functions in the API layer are performing a license check, so I'm guessing we don't need one.